docs/examples/examples.html

```diff
@@ -1,91 +1,1576 @@
-<tr><td><img src="http://higlass.io/t/?d=HncLr3ySQx6t_AGlqN0zhA" width="150"/></td><td><a href="http://higlass.io/app/?config=HncLr3ySQx6t_AGlqN0zhA">One overview and three detail views</a></td><td></td><td></td><td></td><td>X</td><td></td><td></td><td>X</td><td></td><td></td><td></td><td></td><td>X</td><td></td><td>X</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>X</td></tr>
+      <tr>
+        <td></td>
+        <th><div>2d-chromosome-grid</div></th>
+        <th><div>2d-rectangle-domains</div></th>
+        <th><div>chromosome-grid</div></th>
+        <th><div>combined</div></th>
+        <th><div>cross-rule</div></th>
+        <th><div>divided</div></th>
+        <th><div>heatmap</div></th>
+        <th><div>horizontal-1d-annotations</div></th>
+        <th><div>horizontal-1d-heatmap</div></th>
+        <th><div>horizontal-2d-rectangle-domains</div></th>
+        <th><div>horizontal-bar</div></th>
+        <th><div>horizontal-chromosome-labels</div></th>
+        <th><div>horizontal-gene-annotations</div></th>
+        <th><div>horizontal-heatmap</div></th>
+        <th><div>horizontal-line</div></th>
+        <th><div>horizontal-point</div></th>
+        <th><div>horizontal-stacked-bar</div></th>
+        <th><div>osm-tiles</div></th>
+        <th><div>vertical-1d-annotations</div></th>
+        <th><div>vertical-1d-heatmap</div></th>
+        <th><div>vertical-bar</div></th>
+        <th><div>vertical-chromosome-labels</div></th>
+        <th><div>vertical-gene-annotations</div></th>
+        <th><div>vertical-line</div></th>
+        <th><div>viewport-projection-center</div></th>
+      </tr>
+      <tr>
+        <td>local</td>
+      </tr>
+      <tr>
+        <td>
+          <a href="apis/svg.html?/viewconfs/diagonal-split-heatmap.json"
+            >diagonal-split-heatmap.json</a
+          >
+        </td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td>😍</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <a href="apis/svg.html?/viewconfs/1d-annotations.json"
+            >1d-annotations.json</a
+          >
+        </td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td>X</td>
+        <td></td>
+        <td></td>
```

Comment:
I understand that this is just a code style change and not a big deal but maybe a separate issue can be added to change this table so that it is generated in javascript based on some more manageable data structure?

Reply:
I think that might actually already exist. @mccalluc do you happen to remember?

Reply:
This is generated in https://github.com/higlass/higlass/blob/develop/docs/examples/regenerate.py ... I'm not sure it'd be worth while to just change it to a JSON representation: The real issue that it's built artifact which is checked in, and JSON would be the same in that regard... if we wanted to address this, perhaps reference regenerate.py from https://github.com/higlass/higlass/blob/develop/docs/build.sh?

Reply:
Ok, yes probably not worth while. I will mark as resolved.